framework/src/araui/hal/hal_esplcd_impl.c

```diff
@@ -0,0 +1,640 @@
+/****************************************************************************
```

Comment:
1. TizenRT doesn't define hal for lcd h/w. How about integrate lcd interface from nuttx
2. I think here's no specific code related to esp's lcd module. will you implement this kind of impl file for other device in future?
3. Driver should be registered in booting phase like a flash, each of bus., not framework code.